vscode/src/createProject.ts

```diff
@@ -149,4 +165,199 @@ export async function initProjectCreator(context: vscode.ExtensionContext) {
       },
     ),
   );
+
+  type LocalProjectRef = {
+    path: string; // Absolute or relative path to the project dir
+  };
+
+  type GitHubProjectRef = {
+    github: {
+      owner: string;
+      repo: string;
+      ref: string;
+      path?: string; // Optional, defaults to the root of the repo
+    };
+  };
+
+  type Dependency = LocalProjectRef | GitHubProjectRef;
+
+  // TODO: Replace with a list of legitimate known Q# projects on GitHub
```

Comment:
Should we remove this before merging it into `main`?

Reply:
Before release, but not before DEV build. (That was also Stefan's comment I dismissed).